readme.md

```diff
@@ -26,7 +31,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+We use the .post method of jQuery to submit a form via AJAX so that we do not have to do a page refresh.
```

Comment:
We were looking more for the process of how (collect data from the form, package it up into an object, make a POST request), rather than the how. But the AJAX post part you mentioned is correct.